src/views/articles.hbs

```diff
@@ -1,17 +1,23 @@
 
 {{>searchbar}}
 
-<div id="searchResults" class="search__results">
+<div id="searchResults"
+    class="search__results mw7 center">
   {{#each articles}}
-    <img src={{image}}></img>
-    <h3>{{headline}}</h3>
-    <p>{{trailText}}</p>
-      <form class"article_form" action="/visualize" method="POST">
+  <div class="bg-white pa4 center br3 mb4">
+    <div class="mw9 measure center">
+    <img class="br2" src={{image}}></img>
+    </div>
+    <div class="measure center">
+    <h3 class="f4 f2-ns mt2 mb3">{{headline}}</h3>
+    <p class="f5 tj">{{trailText}}</p>
+    <form class"article_form" action="/visualize" method="POST">
       <input name="articleId" type="hidden" value={{id}}>
-      <input class="article__btn" name="articleID" type="submit" value="Visualize">
-
-
-
+      <input class="article__btn
+      input__reset bn shadow-1 bg-light-blue br3
+      pa3 f5 white fw6 center" name="articleID" type="submit" value="Visualize">
```

Comment:
It would improve usability to add a hover effect wherein the cursor changes to a pointer when I hover over this button (I'll let you find that in the tachyons docs)